Packs/CommonScripts/Scripts/URLSSLVerification/URLSSLVerification.py

```diff
@@ -30,48 +31,79 @@ def arg_to_list_with_regex(arg):
     return arg
 
 
+def verify_ssl_certificate(url):
```

Comment:
```suggestion
def verify_ssl_certificate(url: str):
```
Also return type hint